### DISCUSSION TITLE
My Chia account do not get Chia after withdrawing from mining pool H9

### DISCUSSION THREAD

**Discussion by Tiger-Stone** [Chia-Network/chia-blockchain#16705] [Support]
Hi,
My Chia account does not get Chia after withdrawing from mining pool H9 using the same Chia address used for previous withdraws.
H9 system shows that transaction (10.332 CHINA) has been confirmed on 18 October 2023, but my China account has not received yet.
Please help.
Thanks,

**Comment by BrandtH22** [comment]
Hey @Tiger-Stone , as a heads up we can generally provide more thorough and timely support in our discord server (https://discord.gg/chia).

Something I would try a resync of the wallet:
- If using the gui go to settings -> advanced -> resync . This will close the client and upon reopening it will resync. 
- If using the cli then I would stop chia, rename the wallet db folder in `~\.chia\mainnet\wallet\db` then restart chia.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
